x/group/client/testsuite/cli_test.go

```diff
@@ -0,0 +1,2047 @@
+package testsuite
```

Comment:
2k lines for a file is becoming hard to navigate. How about splitting into 2 tests files: txs and queries?